salt/repo/map.jinja

```diff
@@ -0,0 +1,18 @@
+{%- set defaults = salt['grains.filter_by']({
+  'Linux': {
+    'online_mode': True
+  }
+}, grain='kernel', merge=salt['pillar.get']('repo')) %}
+
+{%- set repo = salt['grains.filter_by']({
+  'RedHat': {
+    'containerd': {
+      'name': 'epel'
+    },
+    'kubernetes': {
+      'name': 'kubernetes'
+    }
+  },
+  'Debian': {
```

Comment:
Not supported, so let's not mention it, if possible.

Reply:
True